packages/formatter-html/src/assets/js/scan/get-message.js

```diff
@@ -0,0 +1,34 @@
+// Autogenerated by scripts/create-get-message.js
+
+(function () {
```

Comment:
Same question here as to whether we need to commit this generated file.

Reply:
Same response as before :)